### DISCUSSION TITLE
Progressbar

### DISCUSSION THREAD

**Discussion by JoakimThulin** [stefansundin/privatkopiera#199] [Ideas]
Hej, skulle du kunna tänka dig att  lägga till '-v quiet -stats' på kommandoraden, så att den blir i stil med ffmpeg -v quiet -stats -i "...". På så sätt får man tydligt framdriften vid kommandoprompten

**Comment by stefansundin** [comment]
Bra idé. Istället för att lägga till exakt de argumenten så har jag lagt till ett alternativ i e61d8ba1695ad9aa4b98f3f6d1352c84616971a2 där man kan ändra på FFmpeg-sökvägen. Normalt så använder man det till att ändra `ffmpeg` till något annat, men det är inget som hindrar användaren att lägga till extra argument istället.

Kommer i v0.4.0.

**Reply by JoakimThulin** [reply]
Har testat i 0.41, funkar perfekt! Tack så mycket.
